python/test_dis

```diff
@@ -14,15 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from veles.dis.isa.falcon import FalconIsa
-from veles.data.bindata import BinData
 import sys
+from pprint import pprint
+from veles.data.bindata import BinData
```

Comment:
If you are so strict about fixing nits include an empty line between stdlib and local imports :)